src/field/index.jsx

```diff
@@ -38,12 +38,15 @@ class Field extends Component {
         <h2>{ title }</h2>
         {
           this.props.editable && (!this.state || this.state.editing)
-            ? <TextArea
-              label=""
-              name={name}
-              value={this.state ? this.state.content : content}
-              onChange={this.onChange}
-            />
+            ? <Fragment>
+              <ReactMarkdown>{ content }</ReactMarkdown>
+              <TextArea
+                label=""
+                name={name}
+                value={this.state ? this.state.content : content}
+                onChange={this.onChange}
+              />
+            </Fragment>
```

Comment:
You can get rid of the `Fragment` and the ternary by pulling the `<ReactMarkdown>` bit up outside the conditional.

Reply:
you certainly can

Reply:
I will